dace/frontend/python/replacements.py

```diff
@@ -4371,3 +4406,83 @@ def _inv(pv: 'ProgramVisitor', sdfg: SDFG, state: SDFGState, inp_op: str):
     state.add_memlet_path(chlsky_node, out, src_conn="_b", memlet=Memlet.from_array(*out_arr))
 
     return out_arr[0]
+
+
+# CuPy replacements
```

Comment:
Since CuPy is supposed (?) to follow NumPy syntax, maybe we should make a wrapper around all current NumPy replacements and automatically support the CuPy versions instead of writing new replacements? I would leave this for another PR though.

Reply:
Agreed, added to the [Frontend Fixes](https://github.com/spcl/dace/projects/10) project as a task.